### DISCUSSION TITLE
OpenHD VS Wifibroadcast

### DISCUSSION THREAD

**Discussion by glory86** [svpcom/wfb-ng#226] [General]
hello Vasily

i don't know much a bout openhd
but i can see that it depends on wifibroadcast, atleast for the older versions 

1.  can you as a professional in this field and someone who knows both of them under the hood
to explain what are the technical differences between them

2. i see that openhd moved almost entirely to c++ after version 2.0.8
does that mean it is no longer depending on wifibroadcast

**Comment by svpcom** [comment]
Old versions of openhd use some parts of my wifibroadcast for telemetry link. But video link use original ez-wifibroadcast. New version of openhd (but it is not released yet as I know) will use most of ideas of my wifibroadcast. The main differences are:
- WFB use modular unix-style design with several independent levels of abstraction while OpenHD has monolithic design
- WFB use data encryption for all streams
- WFB provides genetic ipv4 tunnel (you can ssh to drone for example)
- OpenHD has video resolution switch logic and UI for configuration (WFB use plain text config files)
- OpenHD has advanced OSD.  WFB has basic OSD
- WFB can be easily ported to any hw arch (I use it on many arm boards and x86 as ground station). OpenHD can work only on Raspberry PI and Nvidia jetson

So OpenHD is complete solution for hobby use when WFB is more developer friendly and professional use ready.
